pkg/agent/client/client_test.go

```diff
@@ -696,6 +824,19 @@ func TestFetchJWTSVID(t *testing.T) {
 			},
 			err: "JWTSVID issued after it has expired",
 		},
+		{
+			name: "grpc call to NewJWTSVID fails",
```

Comment:
Can we also test that there is an error level log with the expected error fields?